sulley/sessions.py

```diff
@@ -551,8 +551,13 @@ def error_handler (e, msg, target, sock=None):
             # if fuzzing is not finished, web interface thread will catch it
             if self.total_mutant_index == self.total_num_mutations:
                 import signal
-                while True:
-                    signal.pause()
+                try:
+                    while True:
+                        signal.pause()
+                except AttributeError:
+                    # signal.pause() is missing for Windows; wait 1ms and loop instead
+                    while True:
+                        time.sleep(0.001)
```

Comment:
I originally poked around the web for something better, but this is all I found. After looking again just now, I found a few more duct tape solutions, things like waiting for dummy threading events. Nothing that seemed worth it IMO.